### DISCUSSION TITLE
When deleting an issue that was created/linked from a task list, the name seems to persist in the task list

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4289] [Projects and Issues]
I created a task list, converted one of the items to an issue, then deleted the issue. The entry in my task list seemed to still mention the issue + the title/etc. I didn't wait around to see if this would 'eventually correct/update itself', so it might just be an 'eventually consistent thing. Figured I would raise it here in case.

**Comment by labrose** [comment]
I tried to reproduce this behavior by:

1. Opening an issue
2. Adding a task list of a couple text items
3. Converting one of the items to an issue
4. Deleting that issue
5. Going back to the main tracking issue

The task list still includes the deleted issue's id as plain text (like `- [ ] #123`) but the issue title is no longer unfurled because the issue does not exist. To clarify, is that what you were expecting to happen, or something else?

**Reply by 0xdevalias** [reply]
That’s what I would have expected to happen. So must be something weird going on on my side. Might by a browser extension I have such as GitHub refined interfering

**Reply by labrose** [reply]
Could be some live update bug too - what browser/os combo are you using?

**Reply by 0xdevalias** [reply]
Fairly recent version of Chrome (not latest) on latest macOS. Can get specifics when I’m on computer next.
